helm/values.yaml

```diff
@@ -150,4 +162,32 @@ agent:
           "app_signals": { }
         }
       }
-    }
\ No newline at end of file
+    }
+  enableGpuConfig: 
+    {
+      "logs": {
+        "metrics_collected": {
+          "kubernetes": {
+            "gpu_metrics": true
+          },
+        }
+      }
+    }
+
+dcgmExporter:
+  name:
+  image:
+    repository: nvcr.io/nvidia/k8s/dcgm-exporter
```

Comment:
I am assuming this will be our mirrored repo eventually

Reply:
correct